src/common/redis_module/ray_redis_module.cc

```diff
@@ -717,23 +696,77 @@ int TableAppend_RedisCommand(RedisModuleCtx *ctx,
     // necessary since we implement the log with a sorted set, so all entries
     // must be unique, or else we will have gaps in the log.
     RAY_CHECK(flags == REDISMODULE_ZADD_ADDED) << "Appended a duplicate entry";
-    // Publish a message on the requested pubsub channel if necessary.
-    TablePubsub pubsub_channel = ParseTablePubsub(pubsub_channel_str);
-    if (pubsub_channel != TablePubsub::NO_PUBLISH) {
-      // All other pubsub channels write the data back directly onto the
-      // channel.
-      return PublishTableAdd(ctx, pubsub_channel_str, id, data);
-    } else {
-      return RedisModule_ReplyWithSimpleString(ctx, "OK");
-    }
+
```

Comment:
I think you need to reply with a Redis error using `RedisModule_ReplyWithError` here.

Reply:
Where?  If you mean `    RAY_CHECK(flags == REDISMODULE_ZADD_ADDED) << "Appended a duplicate entry";` then I think no since (1) it will crash (2) the existing behavior is like so too?